src/test/java/com/google/step/youtube/YoutubeInfoScraperTest.java

```diff
@@ -32,6 +32,10 @@
 import com.google.api.services.youtube.YouTube.Channels;
 import com.google.api.services.youtube.YouTube.PlaylistItems;
 import com.google.api.services.youtube.model.PlaylistItem;
+import com.google.api.services.youtube.model.PlaylistItemSnippet;
+import com.google.api.client.util.DateTime;
+import java.util.Date;
+import com.google.api.services.youtube.model.ResourceId;
 import java.util.List;
 import java.io.IOException;
 import org.junit.Assert;
```

Comment:
Remove unused imports: Assert, JUnit4, and Mock

Reply:
Done.